app/resources/v1/user_session.js

```diff
@@ -1,4 +1,4 @@
-const User = require('../../models/user.js')
+const { User } = require('../../db/models')
 const logger = require('../../../lib/logger.js')()
 
 exports.delete = async function (req, res) {
```

Comment:
See comment above about retaining use of this API endpoint over exporting the `logout` "verb" from users.